README.md

```diff
@@ -14,7 +14,6 @@ To reformat files you can use:
 ```bash
 mvn spotless:apply
 ```
-or load `alfresco-formatter.xml` to IntelliJ
```

Comment:
Is this no good?

Reply:
I felt it would be better to move info about formatter as close to the `Code Quality` header as possible since most people would like to just import it without having to know anything about spotless